playbooks/roles/heketi-gluster-server/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: create gluster kubenow yaml directory
+  file:
+    path: "~/.kubernetes-yaml/glusterfs/"
+    state: directory
+    mode: 0755
+
+- name: copy gluster DaemonSet configuration
+  copy:
+    src: glusterfs-daemonset.yml
+    dest: "~/.kubernetes-yaml/glusterfs/glusterfs-daemonset.yml"
+
+- name: create gluster DaemonSet
+  command: >
+    kubectl apply -f
+    $HOME/.kubernetes-yaml/glusterfs/glusterfs-daemonset.yml
+
+# the plays below is not really needed - just for verifying everything is OK
```

Comment:
What I mean is that sometimes we turn the deployment into more serial than what is needed with kubernetes, we are waiting for everything to come up before moving on, when more tasks could be parallelllized.